README.md

```diff
@@ -548,6 +548,15 @@ lookup("<sourceField>", "<mapFile>", sep_char: ”,”)
 lookup("<sourceField>", "<mapName>")
 lookup("<sourceField>", "<mapName>", default: "NA")
 ```
+#### `lookup_rdf`
```

Comment:
Heading has wrong level, empty line missing above.

Reply:
right! Fixed.

Reply:
Thanks, but still missing an empty line above the heading I'm afraid ;)

Reply:
fixed